continuous_load_testing/client.go

```diff
@@ -263,6 +284,39 @@ func ExecuteHalfDuplexCalls(ctx context.Context, tc test.TestServiceClient) erro
 	return nil
 }
 
+func ExecuteStreamedSequentialUnaryCall(ctx context.Context, tc test.TestServiceClient) error {
+	for {
+		stream, err := tc.StreamedSequentialUnaryCall(ctx)
+		if err != nil {
+			log.Printf("Failed to create StreamedSequentialUnaryCall stream: %v. Retrying...", err)
+			continue
+		}
+		log.Println("StreamedSequentialUnaryCall stream established.")
+
+		for {
+			_, err := stream.Recv()
```

Comment:
It needs to be in a separate go routine. Please check https://grpc.io/docs/languages/go/basics/#bidirectional-streaming-rpc-1